sqldelight-compiler/integration-tests/src/test/kotlin/com/example/QueryWrapper.kt

```diff
@@ -14,11 +14,11 @@ class QueryWrapper(
 
     val playerQueries: PlayerQueries = PlayerQueries(this, database)
 
-    companion object Helper : SqlDatabase.Helper {
+    companion object Schema : SqlDatabase.Schema {
```

Comment:
i dont think it makes sense for this to be a companion object anymore. Can we make it a normal named inner object, and then have a nameless memberless companion object so people can extend if they want?